docker-compose.test.yml

```diff
@@ -3,7 +3,7 @@ services:
   redis:
     image: redis:3.2-alpine
   postgres:
-    image: mdillon/postgis:9.6-alpine
+    image: mdillon/postgis:11
```

Comment:
Do we want `mdillon/postgis:11` or `mdillon/postgis:11-alpine`?

Reply:
Honestly i'm not THAT sure it matters. Production is AWS RDS so always going to be slightly different. Alpine is certainly smaller, so 👍